frontend/js/components/ImageIcon/ImageIcon.component.js

```diff
@@ -39,7 +39,7 @@ class ImageIcon extends HTMLElement {
 
     constructor() {
         super();
-        this.addEventListener('click', Actions[this.clickAction]);
+        this.addEventListener('click',function(){ Actions.setModalId(this.modalDataTarget.substring(1))});
```

Comment:
use ES6 notation for functions

Reply:
this function call is not needed hence removed